src/rnode-env.mjs

```diff
@@ -94,3 +99,18 @@ function makeGetDeployResult({ deployService }) {
     return listenData;
   };
 }
+
+/**
+ * @param {Object} arg
```

Comment:
My habit is to call this `io`.

Reply:
What you mean by `io` in this context?

Reply:
I mean:

```suggestion
 * @param {Object} io
```

`deployService` is a form of IO. Most powerful stuff that we pass around is.

Reply:
OK, I can see your perspective as argument that can serve IO operations. I'm looking more as function _input_ **arg**uments.